src/client/NuGet.CommandLine.StagingExtensions/StageCommand.cs

```diff
@@ -0,0 +1,135 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Globalization;
+using System.Threading.Tasks;
+using NuGet.Common;
+using NuGet.Configuration;
+using NuGet.Protocol.Core.Types;
+using NuGet.Protocol.Core.v3;
+
+namespace NuGet.CommandLine.StagingExtensions
+{
+    [Command(resourceType: typeof(StagingResources), commandName: "stage", descriptionResourceName: "StageCommandDescription",
+       MinArgs = 0, MaxArgs = 0, UsageSummaryResourceName = "StageCommandUsageSummary")]
+    public class StageCommand : Command
+    {
+        [Option(typeof(StagingResources), "StageCommandCreateDescription")]
+        public string Create { get; set; }
+
+        [Option(typeof(StagingResources), "StageCommandDropDescription")]
+        public string Drop { get; set; }
+
+        [Option(typeof(StagingResources), "StageCommandSourceDescription")]
+        public string Source { get; set; }
+
+        [Option(typeof(StagingResources), "StageCommandApiKeyDescription")]
+        public string ApiKey { get; set; }
+
+        public override async Task ExecuteCommandAsync()
+        {
+            // Verify input
+            if (!string.IsNullOrEmpty(Create) && !string.IsNullOrEmpty(Drop) ||
+                (string.IsNullOrEmpty(Create) && string.IsNullOrEmpty(Drop)))

```

Comment:
done

---
In reply to: [66685931](https://github.com/NuGet/NuGet.Staging/pull/62#discussion_r66685931) [](ancestors = 66685931)